lib/scanner.lpp

```diff
@@ -1758,6 +1759,7 @@ module{WS}+{WORD}                       { const char* mod = trim_inplace(yytext
 \>{WS}*\>{WS}+=                         check_broken_op(yylloc, yytext, ">>="); return SHIFT_RIGHT_EQUALS;
 \<{WS}+\<{WS}*=                         check_broken_op(yylloc, yytext, "<<="); return SHIFT_LEFT_EQUALS;
 \<{WS}*\<{WS}+=                         check_broken_op(yylloc, yytext, "<<="); return SHIFT_LEFT_EQUALS;
+\.{WS}+\.                               check_broken_op(yylloc, yytext, ".."); return RANGE;
```

Comment:
This shouldn't be here... The broken operators above this are left here only because of backwards compatibility... New ones shouldn't allow this.